web-playground/tsconfig.json

```diff
@@ -18,9 +18,17 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "typeRoots": [
+      "./node_modules/@types/"
+    ],
+    "types": [
+      "puppeteer",
+      "jest-environment-puppeteer",
+      "expect-puppeteer"
+    ]
   },
   "include": [
     "src"
   ]
-}
+}
```

Comment:
No EOF `\n` (FWIW)

Reply:
Ha ha, I saw that and half expected a review comment 🙂

Reply:
Apparently I have a formatter that does that on JSON files!

Reply:
Fixed.